apps/core/lib/core/snitch/data/schema/payment/payment.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Core.Snitch.Data.Schema.Payment do
+  @moduledoc """
+  Models a Payment
+  """
+
+  use Core.Snitch.Data.Schema
+
+  @type t :: %__MODULE__{}
+
+  @payment_types ["chk", "ccd"]
+
+  schema "snitch_payments" do
+    field(:slug, :string)
+    field(:payment_type, :string, size: 3)
```

Comment:
Shouldn't we call this _just_ `:type`?

Reply:
yes we can, but type, method might collide with keywords, or may lead to confusion